package_policies/R5.md

```diff
@@ -1,3 +1,9 @@
 **R5.** It is recommended that each package provide a **mechanism to export its ordered list of library
 dependencies** so that any other package or executable linking to the package knows to include these
 dependencies when linking.
+
+The motivation for this policy was that sometimes the order of linking static libraries matters to prevent
+unresolved symbol errors.
+
+The user should have easily accessible information to facilitate linking to the library. This
+information could be provided by a *pkg-config* file, for example.
```

Comment:
In sync with: https://github.com/xsdk-project/xsdk-community-policies/blob/master/package_policies/M7.md - good to use 'RATIONALE:' - or 'MOTIVATION:'?

```suggestion
RATIONALE: When using static libraries,  some compilers require the libraries in the link command to be listed in the correct dependency order - to avoid unresolved symbol errors.

One way this information can  be provided  to the users is via *pkg-config* file.
```